src/app/core/components/widgets/widgetnic/widgetnic.component.html

```diff
@@ -24,18 +24,18 @@
 
               <button mat-icon-button class="" (click)="router.navigate(['network/interfaces/edit/'+nicState.name])">
                 <mat-icon role="img"
-                          matTooltip="Configure {{nicState.name}}"
+                          matTooltip="{{'Configure' | translate}} {{nicState.name}}"
                   matTooltipPosition="above"
-                  aria-label="Configure Interface">
+                  aria-label="{{'Configure Interface' | translate}}">
```

Comment:
the build is complaining about these aria-labels, try removing the translates from these